edit/types.go

```diff
@@ -62,7 +62,11 @@ func IsStringDict(attr string) bool {
 }
 
 // ContainsLabels returns true for all attributes whose type is a label or a label list.
-func ContainsLabels(attr string) bool {
+func ContainsLabels(kind, attr string) bool {
+	if kind == "package_group" && attr == "packages" {
+	  // "package_group" is a special rule and its "packages" attribute is not a list of labels.
```

Comment:
indentation is surprising, make sure you run gofmt

Reply:
I updated IDEA and that disabled autoformatting on save and two spaces look identical to a tabulation. Fixed.